packages/components/src/search-control/index.tsx

```diff
@@ -53,7 +55,9 @@ function SearchControl(
 					label={ __( 'Reset search' ) }
 					onClick={ () => {
 						onChange( '' );
-						searchRef.current.focus();
+						if ( searchRef.current ) {
+							( searchRef.current as HTMLInputElement )?.focus();
```

Comment:
This ref is passed to an `<input>` below, so `searchRef.current` should usually be `HTMLInputElement`

Reply:
No casting needed. See https://github.com/WordPress/gutenberg/pull/42647#discussion_r928627036

Reply:
Thanks! Removed this conditional in https://github.com/WordPress/gutenberg/pull/42647/commits/366b312db28c55cf9694b5021bca421c6065196b